BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/GiniBankNetworkingScreenApiCoordinator.swift

```diff
@@ -492,12 +497,40 @@ extension GiniBankNetworkingScreenApiCoordinator {
                 self.resultsDelegate?.giniCaptureAnalysisDidFinishWith(result: result)
 
                 self.giniBankConfiguration.skontoDiscounts = result.skontoDiscounts
+                self.sendSkontoFeedback(extractions: extractions)
             } else {
                 analysisDelegate?.tryDisplayNoResultsScreen()
                 self.documentService.resetToInitialState()
             }
         }
     }
+    
+    private func sendSkontoFeedback(extractions: [String: Extraction]) {
+        guard let amountToPay = extractAmountToPay(from: extractions) else { return }
+        let amountToPayString = formatAmountToPay(amountToPay)
+        let amountExtraction = createAmountExtraction(value: amountToPayString)
+        self.giniBankConfiguration.sendTransferSummaryWithSkonto(amountExtraction: amountExtraction, amountToPayString: amountToPayString)
```

Comment:
[request] we can remove self from here and move `amountToPayString` to a new line.